roverNYC.json

```diff
@@ -0,0 +1 @@
+{"cloud":{"app_address":"https://app.viam.com:443","id":"85ad1e96-db10-4a84-9e74-acd9b4e2418c","secret":"z34mqjblao47gscs1hlmqv97vpuqk1ofas8qdsxrlvwfqw5y"}}
```

Comment:
[major] remove
don't leak secrets!

Reply:
Indeed; please force-push over this, and delete the robot/remake to invalidate the secret.